pkg/querier/remote_read.go

```diff
@@ -34,7 +35,7 @@ func RemoteReadHandler(q storage.Queryable, logger log.Logger) http.Handler {
 		errors := make(chan error)
 		for i, qr := range req.Queries {
 			go func(i int, qr *client.QueryRequest) {
-				from, to, matchers, err := client.FromQueryRequest(qr)
+				from, to, matchers, err := client.FromQueryRequest(storecache.NewNoopMatcherCache(), qr)
```

Comment:
Why we don't add it here?

Reply:
I just did not want to add another config for queriers for now. WDYT?